### PR TITLE
Add support to update a dev deployment to a prod deployment

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

File: Dockerfile-canary [hunk 1]
```diff
@@ -1,6 +1,7 @@
 FROM ansibleplaybookbundle/apb-base
 # MAINTAINER {{ $MAINTAINER }}
 
+LABEL "com.redhat.apb.version"="1.0"
```
Comment (dymurray) [inline Dockerfile-canary]: This can be removed now that the version is on apb.yml

File: Dockerfile-latest [hunk 2]
```diff
@@ -1,6 +1,7 @@
 FROM ansibleplaybookbundle/apb-base:latest
 # MAINTAINER {{ $MAINTAINER }}
 
+LABEL "com.redhat.apb.version"="1.0"
```
Comment (dymurray) [inline Dockerfile-latest]: Same as above

File: Dockerfile-nightly [hunk 3]
```diff
@@ -1,6 +1,7 @@
 FROM ansibleplaybookbundle/apb-base:nightly
 # MAINTAINER {{ $MAINTAINER }}
 
+LABEL "com.redhat.apb.version"="1.0"
```
Comment (dymurray) [inline Dockerfile-nightly]: Same as above

File: playbooks/update.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+- name: Deploy rhscl-postgresql-apb to openshift
```
Comment (djzager) [inline playbooks/update.yaml]: Just looks like the name of the playbook is off.

**Pull request merged**